l10n_ch_qr_no_amount/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module allows to print QR bills without any amount, so that the customer can define the amount to pay.
+
+Not setting the amount allows to use the same QR code for multiple payments referencing the same invoice.
```

Comment:
Maybe could be stated that how it is done ? with the context key ?

Reply:
I added a bit of explanation how to use it. I don't think it makes sense to explain the technical side in the module's description.